src/EnergyPlus/DataZoneEquipment.cc

```diff
@@ -2274,7 +2272,7 @@ void EquipConfiguration::calcReturnFlows(EnergyPlusData &state,
                 // Establish corresponding airloop inlet(s) mass flow rate and set return node max/min/maxavail
                 Real64 inletMassFlow = 0.0;
                 int maxMinNodeNum = 0;
-                auto &thisAirLoopFlow(state.dataAirLoop->AirLoopFlow(airLoop));
+                auto const &thisAirLoopFlow(state.dataAirLoop->AirLoopFlow(airLoop));
                 if (ADUNum > 0) {
                     // Zone return node could carry supply flow to zone without leaks plus any induced flow from plenum (but don't include other
```

Comment:
[src/EnergyPlus/DataZoneEquipment.cc:1833]:(style),[constVariable],Variable 'zoneNode' can be declared as reference to const
[src/EnergyPlus/DataZoneEquipment.cc:1844]:(style),[constParameter],Parameter 's' can be declared as reference to const
[src/EnergyPlus/DataZoneEquipment.cc:1947]:(style),[constVariable],Variable 'spaceOutletNode' can be declared as reference to const
[src/EnergyPlus/DataZoneEquipment.cc:1982]:(style),[constVariable],Variable 'spaceNode' can be declared as reference to const
[src/EnergyPlus/DataZoneEquipment.cc:2121]:(style),[constVariable],Variable 'thisZoneSysEnergyDemand' can be declared as reference to const
[src/EnergyPlus/DataZoneEquipment.cc:2132]:(style),[constVariable],Variable 'equipOutletNode' can be declared as reference to const
[src/EnergyPlus/DataZoneEquipment.cc:2277]:(style),[constVariable],Variable 'thisAirLoopFlow' can be declared as reference to const